dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -53,23 +55,36 @@
     /**
      * confidential testing config, includes keys
      */
-    public static final String CONFIDENTIAL_CONFIG_PATH = ResourceHelpers.resourceFilePath("dockstoreTest.yml");
+    public static final String CONFIDENTIAL_CONFIG_PATH = getConfidentialConfigPath();
     static final String DUMMY_TOKEN_1 = "08932ab0c9ae39a880905666902f8659633ae0232e94ba9f3d2094cb928397e7";
 
     private CommonTestUtilities() {
 
     }
 
+    private static String getConfidentialConfigPath() {
```

Comment:
In this case, I'd prefer the static initialization block right after CONFIDENTIAL_CONFIG_PATH to emphasize both the `final` and that this shouldn't be called again
https://stackoverflow.com/questions/9379426/java-when-is-a-static-initialization-block-useful